public/scripts/backgrounds.js

```diff
@@ -291,7 +291,7 @@ async function onDeleteBackgroundClick(e) {
     const bgToDelete = $(this).closest('.bg_example');
     const url = bgToDelete.data('url');
     const isCustom = bgToDelete.attr('custom') === 'true';
-    const confirm = await callPopup('<h3>Delete the background?</h3>', 'confirm');
+    const confirm = await callPopup('<h3>' + t`Delete the background?` + '</h3>', 'confirm');
```

Comment:
Let's make this clean:
```suggestion
    const confirm = await Popup.show.confirm(t`Delete the background`, null);
```
(you need to add an import at the top)